modules/Core/pages/register.php

```diff
@@ -353,6 +353,7 @@
                                     'last_online' => $date,
                                     'language_id' => $language_id,
                                     'active' => $active
+                                    'timezone' => ((isset($_POST['timezone']) && $_POST['timezone']) ? Output::getClean($_POST['timezone']) : Output::getClean('Europe/London'))
```

Comment:
as well, on line ~173 in this file, can you add timezone to the validation array?
```php
'timezone' => [
	Validate::TIMEZONE => true
]
```